src/core/utils/nfts.ts

```diff
@@ -48,6 +49,7 @@ export const simpleHashSupportedChainNames = [
   ChainName.proofOfPlayApex,
   ChainName.proofOfPlayBoss,
   ChainName.rari,
+  ChainName.sanko,
   ChainName.scroll,
   ChainName.sei,
   ChainName.xai,
```

Comment:
Looks like SimpleHash hasn't supported these yet, so I think it would cause an API request error: https://docs.simplehash.com/reference/supported-chains-testnets#mainnets

Reply:
I think we should just hit this endpoint instead of defining these manually: https://docs.simplehash.com/reference/chains